src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -189,10 +185,12 @@ private ReplicateActionResponse triggerPostComputeHookOnError(String chainTaskId
                                                                   TaskDescription taskDescription,
                                                                   ReplicateStatus errorStatus,
                                                                   ReplicateStatusCause errorCause) {
-        if (resultService.writeErrorToIexecOut(chainTaskId, errorStatus, errorCause) &&
-                computeManagerService.runPostCompute(taskDescription, "").isSuccessful()) {
+        // log original error
+        logError(errorCause, context, chainTaskId);
+        boolean isOk = resultService.writeErrorToIexecOut(chainTaskId, errorStatus, errorCause);
+        // try to run post-compute
```

Comment:
Can you confirm it only works for standard task so far?

Reply:
Yes, I do.